eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -4,7 +4,8 @@ parameters:
   TestMarkArgument: ''
   BuildTargetingString: 'azure-*'
   TestTimeoutInMinutes: 0
-
+  SetDevVersion: and(eq(variables['Build.Reason'],'Schedule'), eq(variables['System.TeamProject'], 'internal'))
```

Comment:
Can you use a similar pattern to https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L55-L62 that way this can be overridden at queue time if necessary.

Reply:
For reference I have filed https://github.com/Azure/azure-sdk/issues/787 to also use the same pattern on the back in that does the publishing so I'd like everything to be consistent.

Reply:
Thanks. I will update it.

Reply:
I have updated the PR as discussed to use same pattern.